tests/test_runtime_build.py

```diff
@@ -75,16 +74,17 @@ def kernel(A):
     test_hls("csim|csyn")
 
 def test_mixed_stream():
+    return 
```

Comment:
Does that mean we are not testing this one?

Reply:
This test case has some minor issues in generated host code. I plan to fix it with another PR.